client/utils/helpers.js

```diff
@@ -0,0 +1,60 @@
+import jwtDecode from 'jwt-decode';
+
+export const summarize = (str, wordCount = 17) => str
+    .trim()
+    .replace(/\s+/, ' ')
```

Comment:
This does nothing useful, it replaces the first instance of a space character with a space character.